manifests/latest/vsphere-7.0u1/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -107,7 +105,7 @@ spec:
               name: vsphere-config-volume
               readOnly: true
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v2.0.0-rc3
+          image: quay.io/k8scsi/csi-provisioner:v2.0.0
```

Comment:
Same here about adding the new params https://github.com/kubernetes-csi/external-provisioner/blob/master/cmd/csi-provisioner/csi-provisioner.go#L79

Reply:
As mentioned here https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/351#discussion_r487245305 lets keep the default values in this PR.